### PR TITLE
Add setting to clear storage

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Fixes https://github.com/ooni/probe/issues/1271

### CODE REVIEW

File: ooniprobe/AppDelegate.m [hunk 1]
```diff
@@ -19,7 +19,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
         [self copyDBTesting];
         
     [SharkORM setDelegate:self];
-    [SharkORM openDatabaseNamed:@"OONIProbe"];    
+    [SharkORM openDatabaseNamed:@"OONIProbe"];
+    [SharkORM rawQuery:@"PRAGMA journal_mode = DELETE;"];
```
Comment (lorenzoPrimi) [inline ooniprobe/AppDelegate.m:23]: TODO review modes

Comment (lorenzoPrimi) [reply ooniprobe/AppDelegate.m:23]: Done https://blog.devart.com/increasing-sqlite-performance.html

File: ooniprobe/Utility/TestUtility.m [hunk 2]
```diff
@@ -170,6 +170,49 @@ + (BOOL)removeFile:(NSString*)fileName {
...
+/*
+ Return the storage used in the Document folder (in bytes) from .json and .log files indluding db
+ */
++ (uint64_t)storageUsed
+{
+    uint64_t usedSpace = 0;
+    NSString *documentsPath = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) objectAtIndex:0];
+    NSArray *paths = [[NSFileManager defaultManager] contentsOfDirectoryAtPath:documentsPath error:nil];
+    BOOL isDir;
+     for (NSString *path in paths) {
```
Comment (bassosimone) [inline ooniprobe/Utility/TestUtility.m:182]:
```suggestion
    for (NSString *path in paths) {
```

**Review by bassosimone — APPROVED** [review]
🐳

**Pull request merged**